src/atoms/SvgIcon/Hamburger.js

```diff
@@ -9,25 +11,29 @@ const Hamburger = props => (
 		<rect x="0" y="18" width="23" height="5" />
```

Comment:
```suggestion
		<rect x="0" y="18" width="23" height="3" />
```
it will look more purely